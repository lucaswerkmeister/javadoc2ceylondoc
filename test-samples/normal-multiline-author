/**
 * line 1
 *
 * @author some dude
 * @author some other dude
 */
