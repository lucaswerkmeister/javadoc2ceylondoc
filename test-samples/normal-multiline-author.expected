"line 1"
by("some dude", "some other dude")
